### PR TITLE
Different field types for annotation fields

### PR DESCRIPTION
**Pull request by TomDijkema**
Add support for different field types to the annotation wizard form.
Currently supported types:
- String
- Integer
- Boolean
- Select (based on enum)
- Date (with prefilled today date)
- Text (for comments and assessments)

### CODE REVIEW

File: src/components/elements/annotationSidePanel/components/annotationWizard/steps/ExistingInstance.tsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ const ExistingInstance = (props: Props) => {
                                 </Row>
                             </div>
                                 : <div>
-                                    <p className="fs-4"><span className="fw-lightBold">Value:</span> {instanceValue}</p>
+                                    <p className="fs-4"><span className="fw-lightBold">Value:</span> {String(instanceValue)}</p>
```
Comment (southeo) [inline src/components/elements/annotationSidePanel/components/annotationWizard/steps/ExistingInstance.tsx:114]: do you need catch a type exception if `instanceValue` can't be mapped to a string?

File: src/webroot/logos/euFundedLogo.png
Comment (southeo) [inline src/webroot/logos/euFundedLogo.png:1]: :eu:

**Review by southeo — APPROVED** [review]
This was more straightforward than expected :)

File: src/api/sourceSystem/GetSourceSystems.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ const GetSourceSystems = async () => {
     try {
         const result = await axios({
             method: "get",
-            baseURL: `${window.location.protocol}//${window.location.hostname}${window.location.port ? ':' + window.location.port : ''}/api/v1`.replace('dev', 'dev-orchestration').replace('sandbox', 'orchestration'),
+            baseURL: `${window.location.protocol}//${window.location.hostname}${window.location.port ? ':' + window.location.port : ''}/api`.replace('dev', 'dev-orchestration').replace('sandbox', 'orchestration'),
```
[the remark this answers is not in the document]
Comment (TomDijkema) [reply src/api/sourceSystem/GetSourceSystems.ts:21]: Its now handled in the API files as its not part of the base url any more

File: src/components/elements/annotationSidePanel/components/annotationWizard/steps/ExistingInstance.tsx (hunk 1 above)
Comment (TomDijkema) [reply src/components/elements/annotationSidePanel/components/annotationWizard/steps/ExistingInstance.tsx:114]: Instance value should always be convertable to a string, begin a simple data type, the complicated data types are filtered in the step before

**Pull request merged**